reference/docs-conceptual/samples/Collecting-Information-About-Computers.md

```diff
@@ -119,29 +121,30 @@ Name                  :
 Status                :
 CSName                :
 FixComments           :
-HotFixID              : KB4533002
+HotFixID              : KB5030651
 InstalledBy           :
 ServicePackInEffect   :
 PSComputerName        :
 CimClass              : root/cimv2:Win32_QuickFixEngineering
-CimInstanceProperties : {Caption, Description, InstallDate, Name…}
+CimInstanceProperties : {Caption, Description, InstallDate, Name...}
 CimSystemProperties   : Microsoft.Management.Infrastructure.CimSystemProperties
 ...
 ```
 
-The additional data is returned, because the **Property** parameter in `Get-CimInstance` restricts
+The additional data is returned because the **Property** parameter in `Get-CimInstance` restricts
 the properties returned from WMI class instances, not the object returned to PowerShell. To reduce
 the output, use `Select-Object`:
 
 ```powershell
-Get-CimInstance -ClassName Win32_QuickFixEngineering -Property HotFixId |
-    Select-Object -Property HotFixId
+Get-CimInstance -ClassName Win32_QuickFixEngineering -Property HotFixId | Select-Object -Property HotFixId
```

Comment:
Please do not put all of the code on one line. The code was formatted this way for a reason.

Reply:
How  do I edit again